src/services/server/ens/AddressENSService.ts

```diff
@@ -14,8 +14,8 @@ import { SelectAddressENS } from '../../../db/schemas';
 import { ServiceResult } from '../types';
 import { isCartesiUser } from '../utils';
 import { default as Repository } from './AddressENSRepository';
-import { getENSData } from './functions';
-import { AddressEns } from './types';
+import { getENSData, getFreshENSData } from './functions';
+import { AddressEns, ENSAddressData } from './types';
```

Comment:
`ENSAddressData` is not being used, so I suggest we remove it if we don't need it.